src/io/flutter/module/InstallSdkAction.java

```diff
@@ -143,6 +145,7 @@ public void validateSelectedFiles(VirtualFile[] files) throws Exception {
 
       final VirtualFile installTarget = FileChooser.chooseFile(descriptor, null, null);
       if (installTarget != null) {
+        FlutterInitializer.sendAnalyticsAction(getClass().getSimpleName());
```

Comment:
OK, slightly more grounded now :)

The places we're collecting analytics lgtm. Let's rename the actions so we get more semantic events into analytics. I'd suggest `GitCloneAction` => `InstallSdkRunGitAction` and `ViewDocsAction` => `InstallSdkViewDocsAction` or similar. The distinction I'm trying to make is that they're both an attempt to install the sdk - one redirects to git, and one to opening a browser.